dags/main_summary.py

```diff
@@ -29,6 +28,7 @@
 taarlite_cluster_name = "dataproc-taarlite-guidguid"
 taar_locale_cluster_name = "dataproc-taar-locale"
 taar_gcpdataproc_conn_id = "google_cloud_airflow_dataproc"
+taar_dynamo_cluster_name = "dataproc-taar-dynamo"
```

Comment:
Can we give these taar clusters more descriptive names? There are 3 now, and I cant remember what they all do

Reply:
How do you want to name these?  I'm just using the literal job name and prefixing it with `dataproc-` for cluster names.

Reply:
hmm if this cluster is loading dynamo then maybe dataproc-taar-dynamo-loader. Just some descriptor of what the cluster is responsible for doing